axtls/README.md

```diff
@@ -0,0 +1 @@
+Original: Siegmund et al.: [_Attributed Variability Models: Outside the Comfort Zone_](https://www.exastencils.fau.de/pdf/publications/2017/SSA+17.pdf), FSE 2017
```

Comment:
Do you also have a feature model for axTLS? If not: no problem. I intend to extract one this week.